Mlem/Enums/Settings/InternetSpeed.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  InternetSpeed.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2023-08-02.
+//
+
+import Foundation
+
+enum InternetSpeed: String, SettingsOptions {
```

Comment:
Interesting, could we also use this to download lower resolution images from pictrs?

Reply:
I don't see a reason why not, unless pictrs doesn't have the backend support